### PR TITLE
DHFPROD-6734: New example project for a disconnected deployment

### PR DESCRIPTION
**Pull request by rjrudin**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by fsnow — APPROVED** [review]

**Review by SameeraPriyathamTadikonda — CHANGES_REQUESTED** [review]
gradle.properties file missing.

What are the values of dhfGradlePluginDependency and propertiesPluginDependency?

**Comment by rjrudin** [comment]
@SameeraPriyathamTadikonda  Sorry, gradle.properties shouldn't have been needed - I just updated build.gradle to not depend on those properties.

**Review by SameeraPriyathamTadikonda — APPROVED** [review]

File: examples/disconnected-project/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+logic in the project's build.gradle file that configures where Gradle should find plugin dependencies. 
+With a value of "true", the build.gradle file will look in the "gradle-dependencies" directory. Otherwise, 
+it will reach out to remote repositories on the Internet. 
+
+From the hub-deployer directory, you can now deploy your application in the normal fashion. The command 
+below assumes that Gradle will be available in the disconnected environment (i.e. it's using "gradle" instead
+of the Gradle wrapper via "gradlew"). Also, given that the mlUsername and mlPassword properties aren't 
+defined in gradle.properties, they'll need to be defined on the command line:
+
+    ./gradlew -i mlDeploy -PmlUsername=admin -PmlPassword=changeme
```
Comment (pappusowmya) [inline examples/disconnected-project/README.md:49]: gradle wrapper is missing in the zip file that is being created by the task "buildHubDeployer"

Comment (rjrudin) [reply examples/disconnected-project/README.md:49]: That's correct - as noted in the README, the expectation is that the user still has Gradle available in the environment in which they'll be deploying.

**Review by srinathgit — APPROVED** [review]

**Pull request merged**